bluetooth-audio/Dockerfile.template

```diff
@@ -10,17 +10,19 @@ RUN install_packages \
   bluez \
   python-gobject \
   python-dbus \
+  python-gpiozero \
   mplayer
 
-
 # Copy sounds
 COPY sounds /usr/src/sounds
 
-# Setup udev rules - this lets us play the connect/disconnect sound
-# and also turn off discover/pairing when a client is connected
+# Setup udev rules - this lets us play the connect/disconnect sound,
+# turn off discover/pairing when a client is connected
+# and also run a python script
 COPY bluetooth-udev /usr/src/
 RUN chmod +x /usr/src/bluetooth-udev
 COPY udev-rules/ /etc/udev/rules.d/
+COPY udev-python/ /usr/src/
```

Comment:
Last change, this line is outdated, should now be:
`COPY bluetooth-scripts/ /usr/src/bluetooth-scripts/`